Add support for vue-router 5.0.0

npm install is failing with a peer dependency conflict.
I want to update to the new vue-router@5.0.0, but vue-gtag@3.6.3 lists vue-router ^4.5.0 as an optional peer. npm sees that as a conflict and bails.

Full error:

```bash
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: vue-gtag@3.6.3
npm error Found: vue-router@5.0.0
npm error node_modules/vue-router
npm error   vue-router@"^5.0.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peerOptional vue-router@"^4.5.0" from vue-gtag@3.6.3
npm error node_modules/vue-gtag
npm error   vue-gtag@"^3.6.3" from the root project
npm error
npm error Conflicting peer dependency: vue-router@4.6.4
npm error node_modules/vue-router
npm error   peerOptional vue-router@"^4.5.0" from vue-gtag@3.6.3
npm error   node_modules/vue-gtag
npm error     vue-gtag@"^3.6.3" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/package.json
+++ b/package.json
@@ -58,7 +58,7 @@
   "homepage": "https://github.com/MatteoGabriele/vue-gtag#readme",
   "peerDependencies": {
     "vue": "^3.5.13",
-    "vue-router": "^4.5.0"
+    "vue-router": "^4.5.0 || ^5.0.0"
   },
   "devDependencies": {
     "@biomejs/biome": "1.9.4",
@@ -74,7 +74,7 @@
     "typescript": "^5.8.3",
     "vitest": "^3.0.5",
     "vue": "^3.5.13",
-    "vue-router": "^4.5.0"
+    "vue-router": "^4.5.0 || ^5.0.0"
   },
   "peerDependenciesMeta": {
     "vue-router": {

--- a/tsdown.config.ts
+++ b/tsdown.config.ts
@@ -1,9 +1,9 @@
 import { defineConfig } from 'tsdown'
-import { name } from './package.json'
+import pkg from './package.json' with { type: 'json' }
 
 export default defineConfig({
   entry: {
-    [name]: './src/index.ts'
+    [pkg.name]: './src/index.ts'
   },
   platform: 'browser',
   minify: true,

